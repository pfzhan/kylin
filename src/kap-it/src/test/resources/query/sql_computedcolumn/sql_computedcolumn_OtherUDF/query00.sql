

select count(REGEXP_LIKE("LSTG_FORMAT_NAME", '^FP |GTC$')),
       count(REGEXP_LIKE("LSTG_FORMAT_NAME",'FP-GT+C*')),
       count(REGEXP_LIKE("LSTG_FORMAT_NAME",'FP-(non )?GTC')),
       count(REGEXP_LIKE("LSTG_FORMAT_NAME",'FP-[A-Z][^a-z]C'))
from test_kylin_fact