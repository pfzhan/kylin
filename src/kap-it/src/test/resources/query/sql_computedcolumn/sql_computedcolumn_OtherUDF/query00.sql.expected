

select count(rlike("LSTG_FORMAT_NAME", '^FP |GTC$')),
       count(rlike("LSTG_FORMAT_NAME",'FP-GT+C*')),
       count(rlike("LSTG_FORMAT_NAME",'FP-(non )?GTC')),
       count(rlike("LSTG_FORMAT_NAME",'FP-[A-Z][^a-z]C'))
from test_kylin_fact